scripts/create-test-s3-bucket.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# Wait 2 seconds to ensure minio is finished bootstrapping
+# TODO: Update this to do event based polling on minio server directly vs semi-arbitrary timeout
+sleep 2s
+
+# Configure MinIO client (mc)
+echo "Configuring MinIO client..."
+mc alias set local http://minio:9000 minioadmin minioadmin
```

Comment:
What is mc? I don't have it on my system. Is this an external dependency we're adding? Prob want to document it somewhere. Eventually we'll prob want to have a mise file like op (or go all out and use nix)

Reply:
oo probably worth noting that this script is only runnable in the `minio-init` service since it references the docker network from within